etna/analysis/feature_relevance/relevance.py

```diff
@@ -0,0 +1,51 @@
+from abc import ABC
+from abc import abstractmethod
+
+import pandas as pd
+
+from etna.analysis.feature_relevance.relevance_table import get_statistics_relevance_table
+
+
+class RelevanceTable(ABC):
```

Comment:
Shell we also inheritance from `BaseMixin`?

Reply:
fixed